pom.xml

```
@@ -33,8 +33,12 @@
     <selenium.version>4.25.0</selenium.version>
     <!-- aligned with selenium -->
     <guava.version>33.3.0-jre</guava.version>
+
+    <!-- Keep them aligned to Maven version -->
     <maven.version>3.9.6</maven.version>
+    <maven-sisu.version>0.9.0.M3</maven-sisu.version>
```

According to https://github.com/apache/maven/blob/maven-3.9.6/pom.xml#L130 Maven 3.9.6 uses 0.9.0.M2, so I think this should be 0.9.0.M2 in order to follow the comment two lines earlier ("Keep them aligned to Maven version"):

```suggestion
    <maven-sisu.version>0.9.0.M2</maven-sisu.version>
```

Please feel free to ignore this suggestion if I have misunderstood the intention of the comment above.

---

Yup, let be consistent and have them **truly aligned**. Also, checked, and M2 does provide no_asm as well:
https://repo.maven.apache.org/maven2/org/eclipse/sisu/org.eclipse.sisu.inject/0.9.0.M2/

---

In reality does not stir much (as Sisu versions were more important due shaded in Asm, but now you use own ASM), also I don't want to kill current CI jobs, so please merge this  suggestion _before_ merging the PR

---

Oh, 0.9.0.M2 does not have "strict" mode, so if downgrading, this will need change:
remove "false" after INDEX here https://github.com/jenkinsci/acceptance-test-harness/pull/1733/files#diff-14b9e71c9ede5549e786ac9e299d0b2da2969ff999f9c0a9ad55a3849d8abca2R40